File: pset6/mario.py
import cs50

def main():
    height = -1
    while height < 0 or height > 23:
        print("Height: ", end="")
        height = cs50.get_int()

    for l in range(1, height + 1):
        for i in range(height - l):
            print(" ", end="")
        for i in range(height - l, height):
            print("#", end="")
        
        print("  ", end="")
        
        for i in range(height - l, height):
            print("#", end="")

        print("")


if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
    int height = -1;
    while (height < 0 || height > 23) {
        printf("Height: ");
        height = get_int();
    } 
    
    for (int l = 1; l <= height; l++) {
        for (int i = 0; i < height - l; i++)
            printf(" ");
        for (int i = height - l; i < height; i++)
            printf("#");
        
        printf("  ");
        
        for (int i = height - l; i < height; i++)
            printf("#");
        //for (int i = 0; i < height - l; i++)
        //    printf(" ");
        
        printf("\n");
    }
}